src/assessment.py

```diff
@@ -121,3 +124,47 @@ def test_assessment_blank():
         return "response from AI or service not valid", 400
 
     return grades
+
+# Submit a test rubric assessment with examples
+@assessment_routes.route('/test/assessment/examples', methods=['GET', 'POST'])
+def test_assessment_examples():
+    openai.api_key = os.getenv('OPENAI_API_KEY')
+
+    with open('tests/data/u3l13_01.js', 'r') as f:
+        code = f.read()
+
+    with open('tests/data/u3l13.txt', 'r') as f:
+        prompt = f.read()
+
+    with open('tests/data/u3l13.csv', 'r') as f:
+        rubric = f.read()
+
+    examples = []
+    with open('tests/data/example.js', 'r') as f:
+        examples.append(f.read())
+    with open('tests/data/example.tsv', 'r') as f:
+        examples.append(f.read())
+
+    try:
+        grades = assess.grade(
+            code=code,
+            prompt=prompt,
+            rubric=rubric,
+            examples=[examples],
+            api_key=request.values.get("api-key", openai.api_key),
+            llm_model=request.values.get("model", "gpt-4"),
+            remove_comments=(request.values.get("remove-comments", "0") != "0"),
+            num_responses=int(request.values.get("num-responses", "1")),
+            temperature=float(request.values.get("temperature", "0.2")),
+        )
+    except ValueError as e:
+        return "One of the arguments is not parseable as a number: {}".format(str(e)), 400
+    except openai.error.InvalidRequestError as e:
+        return str(e), 400
+    except KeyConceptError as e:
+        return str(e), 400
+    
+    if not isinstance(grades, dict) or not isinstance(grades.get("data"), list):
+        return "response from AI or service not valid", 400
+
+    return grades
```

Comment:
I am still seeing newlines missing at the end of files. we should really have a lint rule to prevent this as it can lead to problems. in the meantime, can you please double check that these are added back everywhere you see this symbol in this PR?

Reply:
I updated the settings and went through the files to fix this. I'll double check this one.

Reply:
Thanks Mark, it does look like you fixed this prior to my comment. I'm not sure how I got to looking at an old diff. What you have in this PR now looks good to me.

Reply:
to be clear I was looking at the 🚫 symbol which github shows at the end of the file